TypeCobol/Compiler/Preprocessor/CopyNameMapFile.cs

```diff
@@ -36,5 +38,17 @@ public bool Contains(string name)
             return Array.BinarySearch(_cpyCopyNames, name.ToUpper()) >= 0;
         }
     }
+
+    /// <summary>
+    /// Helper methods
+    /// </summary>
+    public static class CopyNameMapFileExtensions
```

Comment:
Why not put these methods directly in the related classes ?

Reply:
This allow to factorize the `Contains` logic